Firestore/Source/Public/FIRFirestoreSettings.h

```diff
@@ -19,7 +19,7 @@
 NS_ASSUME_NONNULL_BEGIN
 
 /** Used to set on-disk cache size to unlimited. Garbage collection will not run. */
-extern const int64_t kFIRFirestoreCacheSizeUnlimited;
+extern const int64_t kFIRFirestoreCacheSizeUnlimited NS_SWIFT_NAME(FirestoreCacheSizeUnlimited);
```

Comment:
Could you add a bit to BasicCompileTests.swift that verifies this translates into something sane?

Maybe in `initializeDb`, something like:

```
var cacheSize: Int64 = FirestoreCacheSizeUnlimited
settings.cacheSizeBytes = cacheSize
```

Reply:
It's already there. I have `settings.cacheSizeBytes = FirestoreCacheSizeUnlimited`

Reply:
https://github.com/firebase/firebase-ios-sdk/pull/1925/files#diff-971282f2b8fb688f5ba427c5d190d0fdR67